### DISCUSSION TITLE
AI人工智能雖然很聰明，但在團隊合作協調方面仍有些問題

### DISCUSSION THREAD

**Discussion by EricAgumonJou** [1101-GS4719-C/Python-Community-GS4719-C#147] [General]
在國際象棋或圍棋等遊戲方面，人工智慧 (AI) 已遠遠超過了不少頂尖棋手。這些超人般的人工智慧無與倫比的競爭對手，但與人類合作或許比與人類競爭更難，同樣的技術能和人和諧相處嗎？

有不少研究顯示，人類在玩需要團隊合作的遊戲時，發現人工智慧可能淪為豬隊友，並對“團隊智能”構成了一大挑戰。其中在一項新的研究中，麻省理工學院林肯實驗室的研究人員試圖找出人類在使用先進的人工智慧模型玩需要扣過合作來的紙牌遊戲 Hanabi 時的表現，該模型經過訓練以強化與從陌生隊友一起玩。

在單盲實驗中，參與者玩了兩個系列的遊戲：一個是 AI 代理作為他們的隊友，另一個則是當rule-based agent且手動打code以預先定義遊戲策略的機器人。

最終結果令研究人員感到驚訝，人工智慧隊友的得分竟然比rule-based agent得來得差，而且合作過的人們大多反應討厭他們的人工智慧隊友，他們也認為人工智慧隊友是不可預測的、不可靠的且不可信的，即使團隊有得分，他們仍會感到消極。
![Hanabi_Experiment](https://user-images.githubusercontent.com/91121500/140015237-2481b8d4-26b0-49dc-8a26-6338c7f1876b.png)

資料來源:
https://news.mit.edu/2021/does-artificial-intelligence-play-well-others-1004

**Comment by EricAgumonJou** [comment]
未來若能好好掌握 AI 與人類之間的 Hanabi 之類的遊戲，那麼智慧團隊合作之開闢會有無限的可能。

**Comment by enoch-lightning** [comment]
頗令人意外，因為一般人都會認為人工智慧非常強大。而rule-based agent會勝過AI，可能是因為AI需要更多的資訊學習。

**Comment by Jie2222** [comment]
也有AI團隊合作滿成功的案例
[To hone its collaborative skills, this AI is taking on the world's top video game players](https://www.science.org/content/article/hone-its-collaborative-skills-ai-taking-worlds-top-video-game-players)

**Comment by CCello** [comment]
這個研究結果顯示，人工智慧在與人類進行合作方面仍然存在一些挑戰和限制。雖然人工智慧能夠在一些遊戲中表現出超人的水平，但是當它需要與人類進行合作時，它仍然面臨著一些問題。例如，人工智慧可能缺乏人類的直覺和創造力，而這些能力對於在團隊中順利合作非常重要。此外，人工智慧在處理不確定性和未知情況方面也可能存在問題，這些問題可能會導致人工智慧的表現不穩定。因此，與人類合作對於人工智慧來說可能是一個更大的挑戰，需要進一步的研究和改進。
